waku/v2/node/storage/message/sqlite_store/retention_policy.nim

```diff
@@ -0,0 +1,14 @@
+{.push raises: [Defect].}
+
+import
+  stew/results
+import
+  ../../sqlite
+
+
+type RetentionPolicyResult*[T] = Result[T, string]
```

Comment:
Same query re: other stores making use of these functions and types

Reply:
Check my comment: https://github.com/status-im/nwaku/pull/1141#issuecomment-1243721470